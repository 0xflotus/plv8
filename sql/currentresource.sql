CREATE OR REPLACE FUNCTION test_resource_owner()
RETURNS json
AS $$
  return plv8.execute("SELECT 1")[0]
$$ LANGUAGE plv8;
CREATE TABLE resource_table (col TEXT);

SELECT test_resource_owner();
SELECT test_resource_owner() FROM resource_table;
