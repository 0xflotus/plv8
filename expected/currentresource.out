CREATE OR REPLACE FUNCTION test_resource_owner()
RETURNS json
AS $$
  return plv8.execute("SELECT 1")[0]
$$ LANGUAGE plv8;
CREATE TABLE resource_table (col TEXT);
SELECT test_resource_owner();
 test_resource_owner 
---------------------
 {"?column?":1}
(1 row)

SELECT test_resource_owner() FROM resource_table;
 test_resource_owner 
---------------------
(0 rows)

